jobs/seed_repos.groovy

```diff
@@ -0,0 +1,53 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+name = 'deis-seed-repos'
+repoName = 'seed-repo'
+
+job(name) {
+  description """
+    <p>Runs deis/seed-repo against all of the workflow components.</p>
+  """.stripIndent().trim()
+
+  scm {
+    git {
+      remote {
+        github("deis/${repoName}")
+      }
+      branch('master')
+    }
+  }
+
+  logRotator {
+    daysToKeep defaults.daysToKeep
+  }
+
+  triggers {
+    githubPush()
```

Comment:
This will trigger the job on any commit to the master branch of the `seed-repo` repo -- could be what we had in mind, otherwise, we can remove this if we're thinking this job will be manually kicked off for a given repo.